lucid/hie.yaml

```diff
@@ -0,0 +1,4 @@
+cradle:
+  cabal:
+    - path: "./src"
+      component: "lib:lucid-htmx"
```

Comment:
This makes it so that default extensions defined in the `cabal` file propagate to HLS